src/main/java/uk/gov/hmcts/ccd/domain/service/aggregated/SearchQueryOperation.java

```diff
@@ -52,6 +60,10 @@ public SearchResultView execute(final String view,
             return new SearchResultView(Collections.emptyList(), Collections.emptyList(), NO_ERROR);
         }
 
+        final SearchResult searchResult = getSearchResult(caseType.get(), view);
+
+        addSortOrderFields(metadata, searchResult);
```

Comment:
shouldn't we have added not metadata sort fields to queryParams rather than to metadata?